config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
   root "static_pages#home"
-  get "static_pages/help"
-  get "static_pages/about"
-  get "static_pages/contact"
+  get "/home", to: "static_pages#home"
+  get "/help", to: "static_pages#help"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get  "/signup",  to: "users#new"
```

Comment:
`get  "/signup",  to: "users#new"` kiểm tra lại, có tới 2 space á nhé. 1 space thôi em dzai